.gitignore

```diff
@@ -1 +1,20 @@
+# OS specific auto-generated folders and files
+*.swp
+.DS_Store
+
+# Platform specific tools (use fetch-configlet)
+bin/configlet
+bin/configlet.exe
+
+# IDE folders and files
+.idea
```

Comment:
I feel like this should go into local exclude files since people use different IDEs.

Reply:
I don't really care tbh. I added it because a few other repos have it; I agree that it should be local ignored, but this forces keeping it out of this repo.

Reply:
I also don't really care either, but it also isn't harmful so I'm fine with having it here!